### PR TITLE
Fix wrong project issue

### PR DESCRIPTION
**Pull request by joehan**
### Description
Fixes an issue where the VSCE would execute commands in the directory default project instead of in the selected project.

### CODE REVIEW

File: firebase-vscode/src/core/index.ts [hunk 1]
```diff
@@ -61,7 +61,7 @@ export async function registerCore(
         workspaceFolder, // The workspace folder
         "firebase init dataconnect", // how you name the task
         "firebase init dataconnect", // Shows up as MyTask: name
-        new vscode.ShellExecution(`${settings.firebasePath} init dataconnect`),
+        new vscode.ShellExecution(`${settings.firebasePath} init dataconnect ${currentProjectId.value}`),
```
Comment (fredzqm) [inline firebase-vscode/src/core/index.ts:64]: `-P ${currentProjectId.value}`?

Comment (fredzqm) [reply firebase-vscode/src/core/index.ts:64]: nit: Is `--project` more clear?

**Review by fredzqm — APPROVED** [review]

File: firebase-vscode/src/core/index.ts [hunk 2]
```diff
@@ -61,7 +61,7 @@ export async function registerCore(
         workspaceFolder, // The workspace folder
         "firebase init dataconnect", // how you name the task
         "firebase init dataconnect", // Shows up as MyTask: name
-        new vscode.ShellExecution(`${settings.firebasePath} init dataconnect`),
+        new vscode.ShellExecution(`${settings.firebasePath} init dataconnect --project ${currentProjectId.value}`),
```
Comment (yuchenshi) [inline firebase-vscode/src/core/index.ts:64]: Is this guaranteed to be non-empty when the init button is shown? I thought there's a project selection prompt in the CLI or something

File: firebase-vscode/src/data-connect/terminal.ts [hunk 3]
```diff
@@ -26,7 +27,7 @@ export function runCommand(command: string) {
   // TODO: This fails if the interactive shell is not expecting a command, such
   // as when oh-my-zsh asking for (Y/n) to updates during startup.
   // Consider using an non-interactive shell.
-  terminal.sendText(command);
+  terminal.sendText(`${command} -P ${currentProjectId.value}`);
```
Comment (yuchenshi) [inline firebase-vscode/src/data-connect/terminal.ts:30]: Shall we use the long form `--project` for consistency and readability?

**Pull request merged**